curriculum/js1/4.js

```diff
@@ -6,8 +6,14 @@
  * @returns {string}
  */
 
-const solution = (a, b) => {
-  return 0;
+const solution = (x,b,str= "",i=0) => {
+  if(x === i)
+  {
+   return str;
+  };
+
+  str = b  + str
```

Comment:
you forgot the ; at the end of this line.

Reply:
I see.